src/layouts/govuk.njk

```diff
@@ -5,6 +5,8 @@
 
 {% set logo_link_title = 'Go to the GOV.UK PaaS Admin Tool Homepage' %}
 
+{% block inside_header %} <div class="navigation"><ul><li><a href="/organisations">My Organisations</a></li><li><a href="#">Documentation</a></li><li><a href="#">Support</a></li><li><a href="#">Sign out</a></li><ul>{% endblock %}
```

Comment:
Could we make each `li` a new line? The links do not need to be hash tags :)

Documentation: `https://docs.cloud.service.gov.uk/`
Support: `mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk`
Sign out: `/auth/logout`

Reply:
Yeah of course, this was to have something out for deployment, and thought we had less time.